tests/integration/test_users.py

```diff
@@ -81,21 +81,55 @@ def test_refresh_user(client, test_database, admin_user):
     assert responseRefreshToken.status_code == 200
 
 
-def test_get_user_by_id(client, auth_headers, admin_user):
+def test_get_user_by_id(client, empty_test_db, create_admin_user, valid_header):
     """The get user by id route returns a successful response code."""
-    user = UserModel.find_by_email(admin_user.email)
-    response = client.get(f"/api/user/{user.id}", headers=auth_headers["admin"])
+    user = create_admin_user()
+    response = client.get(f"/api/user/{user.id}", headers=valid_header)
```

Comment:
I think we need to assert that we get back serialized data. Asserting the status codes can be done in auth tests.

Reply:
Ok, will do.